### PR TITLE
Doc updates: Repo and scheduler README

### PR DESCRIPTION
**Pull request by rs-vertica**
@NerdLogic for review (I don't have the option on the right of this window to add you as a reviewer, so @'ing you here)

Updated repo-level and scheduler READMES:
- Added TOC
- Repo overview: summary of important files
- Quickstart: Important commands to config and launch scheduler
- Config and launch scheduler: Comprehensive instructions on how to config and launch scheduler
- Build custom container: Use make build to build a custom scheduler container
- Push to Docker Hub: make push to push to remote repo

NOTE: After revisions, it might make sense to move the following to the Docker Hub README:
- Intro
- Prereqs
- Installation
- Quickstart
- Links to GH repo

### CODE REVIEW

File: vertica-kafka-scheduler/README.md
Comment (NerdLogic) [inline vertica-kafka-scheduler/README.md:253]: "mounts `log` from the current directory into the container's `/opt/vertica/log` so that log messages will be written into that local `log` directory."

File: vertica-kafka-scheduler/README.md
Comment (NerdLogic) [inline vertica-kafka-scheduler/README.md:252]: mounts `vkafka-log-config-debug.xml` from the current directory into the container's `/opt/vertica/packages/kafka/config` directory. This file configures log settings to help troubleshoot scheduler issues.

File: vertica-kafka-scheduler/README.md
Comment (NerdLogic) [inline vertica-kafka-scheduler/README.md:254]: ... to specify the uid that has access to write to the log directory.

File: vertica-kafka-scheduler/README.md
Comment (NerdLogic) [inline vertica-kafka-scheduler/README.md:287]: default = whatever version VERTICA_INSTALL is pointing to.

**Review by NerdLogic — APPROVED** [review]
Just some minor comments.

File: vertica-kafka-scheduler/README.md
Comment (rs-vertica) [reply vertica-kafka-scheduler/README.md:169]: Updated

File: vertica-kafka-scheduler/README.md
Comment (rs-vertica) [reply vertica-kafka-scheduler/README.md:253]: Updated.

File: vertica-kafka-scheduler/README.md
Comment (rs-vertica) [reply vertica-kafka-scheduler/README.md:252]: Updated.

File: vertica-kafka-scheduler/README.md
Comment (rs-vertica) [reply vertica-kafka-scheduler/README.md:254]: Updated.

File: vertica-kafka-scheduler/README.md
Comment (rs-vertica) [reply vertica-kafka-scheduler/README.md:287]: Updated.

**Comment by rs-vertica** [comment]
@NerdLogic I updated the doc per your PR comments. Can you please merge this PR when you have a sec? I do not have the necessary write access to this repo.

File: vertica-kafka-scheduler/README.md
Comment (NerdLogic) [inline vertica-kafka-scheduler/README.md:251]: "the current directory file" sound strange to me.  Should the word "file" be removed?
Should we use "in" or "into" as in the line below?  Or maybe the word "as" would work if we changed "mount" to "map".  Something like this might read better:
`maps <file> from the current directory as <file> in the container`

File: vertica-kafka-scheduler/README.md
Comment (NerdLogic) [inline vertica-kafka-scheduler/README.md:5]: well, technically both the pre-built and locally built images are based on alpine:3.14

File: vertica-kafka-scheduler/README.md
Comment (NerdLogic) [inline vertica-kafka-scheduler/README.md:253]: `log` not `/log` on both lines here.  Saying "/log from the current directory" doesn't make sense because / is the root directory.

**Review by NerdLogic — CHANGES_REQUESTED** [review]
Some more comments before merging...

File: vertica-kafka-scheduler/README.md
Comment (rs-vertica) [reply vertica-kafka-scheduler/README.md:5]: removed second "pre-built", so it just says "The image is based on ..."

File: vertica-kafka-scheduler/README.md
Comment (rs-vertica) [reply vertica-kafka-scheduler/README.md:253]: ahhh yeah gotcha--thanks!

File: vertica-kafka-scheduler/README.md
Comment (rs-vertica) [reply vertica-kafka-scheduler/README.md:251]: yes, I think I should have left 'file' out--will remove.

So, for 'mount' --> 'map', I think it might be better to use 'mount' because the '-v' flag is the docker bind mount command, and it might make sense to keep the language 'official' or consistent? Maybe this is overthinking/incorrect, but I kinda see mapping as an association btwn two existing files, rather than copying an existing host file to a different location.

Either way, I revised the sentence--if you still think 'mapping' makes better sense after reading this comment and the revision, we can use 'mapping'.

File: vertica-kafka-scheduler/README.md
Comment (rs-vertica) [reply vertica-kafka-scheduler/README.md:253]: I also changed the wording slightly--let me know if you think it is better or I should revert.

File: vertica-kafka-scheduler/README.md
Comment (NerdLogic) [inline vertica-kafka-scheduler/README.md:252]: Now I'm being picky, but I'd like to see these two mount sentences look the same grammatically.  The first sentence is about mounting a file and the second is about mounting a directory, so they aren't identical, but they should be closer wrt "into", "the container's", "as the", etc.

And as long as you are making changes here, the "This writes log messages" should maybe be "This allows log messages to be written to" ... "assuming `vkafka-log-config-debug.xml` still configures log output in `/opt/vertica/log`

**Review by NerdLogic — CHANGES_REQUESTED** [review]

File: vertica-kafka-scheduler/README.md
Comment (rs-vertica) [reply vertica-kafka-scheduler/README.md:252]: I thought the same thing about the sentence structure, but thought maybe it was more clear bc they were mounting different things. I updated the structure to be the same, let me know what you think.

Comment (NerdLogic) [reply vertica-kafka-scheduler/README.md]: I like it.  Thanks!

**Review by NerdLogic — APPROVED** [review]

**Pull request merged**
